### DISCUSSION TITLE
windows10 cannot start

### DISCUSSION THREAD

**Discussion by roddy20** [utmapp/UTM#6238] [Windows]
Windows 10 cannot start

**Configuration** 
* UTM Version: 4.4.5 (94)
* macOS Version: 14.4 (23E214)
* Mac Chip (Intel, M1, ...): M1

**Crash log**  
<img width="1022" alt="Screenshot 2024-03-31 at 09 09 44" src="https://github.com/utmapp/UTM/assets/5681302/b6161719-2233-4ca1-8823-991e0b6ea3f4">

**Debug log**

**Comment by osy** [comment]
https://docs.getutm.app/guides/windows/#installer-crashes-with-bsod-system_thread_exception_not_handled

**Reply by roddy20** [reply]
Thanks for the answer
I re-reading the instruction (newer win10 iso) and going further a little
<img width="1025" alt="Screenshot 2024-03-31 at 12 15 30" src="https://github.com/utmapp/UTM/assets/5681302/6d542ddb-4625-4c5a-b388-0776ea088bf5">

**Reply by roddy20** [reply]
It seems to me, windows itself is running…virtual machine cannot connect to internet for updates

**Reply by roddy20** [reply]
OOBE\BYPASSNRO not found
and it is Win10 not 11

**Reply by roddy20** [reply]
I read the article again and tried all the described there 
<img width="1021" alt="Screenshot 2024-04-08 at 17 48 49" src="https://github.com/utmapp/UTM/assets/5681302/8f3a9497-9861-4d3b-b7d5-97a1a1b78f0f">
<img width="1025" alt="Screenshot 2024-04-08 at 18 12 27" src="https://github.com/utmapp/UTM/assets/5681302/2f2fd44d-1567-4271-84bd-b0c1d9148f35">

**Reply by roddy20** [reply]
PS tried Win10 Home and Pro from the same ISO
19045.3803.231204-0204.22h2_release_svc_refresh_CLIENTCONSUMER_RET_x64FRE_en-us
no difference

**Reply by sun12yyds** [reply]
建议：重新下载iso映像
